src/App.test.js

```diff
@@ -1,15 +1,22 @@
 import React from "react";
-import '@testing-library/jest-dom/extend-expect'
+import "@testing-library/jest-dom/extend-expect";
 import { render, fireEvent } from "@testing-library/react";
 import { getByTestId, getAllByTestId } from "./utils/test-utils.js";
+import { createMemoryHistory } from "history";
+import { Router } from "react-router-dom";
 
 // component to be tested.
 import App from "./App.js";
-import Login from "../src/components/auth/login";
```

Comment:
This was rearranged because of the order of the file. It made it easier for me to expect what was tested first.